apis/goharbor.io/v1alpha2/harborcluster_types.go

```diff
@@ -114,16 +114,18 @@ type Storage struct {
 // StorageRedirectSpec defines if the redirection is disabled.
 type StorageRedirectSpec struct {
 	// Disable the redirect.
-	// Default is false
-	// +optional
-	Disable bool `json:"disable"`
+	// Default is true
+	// +kubebuilder:default:=true
+	Enable bool `json:"enable,omitempty"`
+	// +kubebuilder:validation:Optional
+	Expose *HarborExposeComponentSpec `json:"expose,omitempty"`
 }
 
 type MinIOSpec struct {
 	harbormetav1.ImageSpec `json:",inline"`
 
 	// Determine if the redirection of minio storage is disabled.
-	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Required
```

Comment:
```suggestion
	// +kubebuilder:validation:Optional
```

I think it's better to keep this item as an optional item.

But at the backend logic, we can keep the default behavior is `enable` the redirect if it is not set here.